packages/core/src/admin-ui/system/adminMetaSchema.ts

```diff
@@ -187,9 +203,12 @@ const KeystoneAdminUIListMeta = graphql.object<ListMetaRootVal>()({
     fields: graphql.field({
       type: graphql.nonNull(graphql.list(graphql.nonNull(KeystoneAdminUIFieldMeta))),
     }),
+    groups: graphql.field({
+      type: graphql.nonNull(graphql.list(graphql.nonNull(KeystoneAdminUIFieldGroupMeta))),
+    }),
     initialSort: graphql.field({ type: KeystoneAdminUISort }),
     ...contextFunctionField('isHidden', graphql.Boolean),
-    isSingleton: graphql.field({ type: graphql.Boolean }),
+    isSingleton: graphql.field({ type: graphql.nonNull(graphql.Boolean) }),
```

Comment:
This is an unrelated I noticed when re-generating the types for the admin meta query, I don't think it was intentionally nullable. Types elsewhere assumed it was non-nullable so I've made it non-nullable.